Get error while install shlink

* Shlink Version: 2.1.3
* PHP Version: 7.4.5
* How do you serve Shlink: Self-hosted Apache
* Database engine used: MariaDB(10.1.44)

#### Summary

I follow the steps from https://shlink.io/documentation/install-dist-file/
when I run bin/install, I provide the information, but get error

```

 Welcome to Shlink!!
 This tool will guide you through the installation process.

DATABASE
========

 Select database type [MySQL]:
  [0] MySQL
  [1] MariaDB
  [2] PostgreSQL
  [3] MicrosoftSQL
  [4] SQLite
 > 1

 Database name [shlink]:
 >

 Database host [localhost]:
 >

 Database port [3306]:
 >

 Database username:
 > shlink

 Database password:
 > <hide>

URL SHORTENER
=============

 Default domain for generated short URLs:
 > <hide>

 Select schema for generated short URLs [http]:
  [0] http
  [1] https
 > https

 Do you want to validate long urls by 200 HTTP status code on response? (yes/no) [yes]:
 >

 What is the default length you want generated short codes to have? (You will still be able to override this on every created short URL) [5]:
 >

REDIRECTS
=========

 Custom URL to redirect to when a user hits Shlink's base URL (If no value is provided, the user will see a default "404 not found" page):
 >

 Custom URL to redirect to when a user hits an invalid short URL (If no value is provided, the user will see a default "404 not found" page):
 >

 Custom URL to redirect to when a user hits a not found URL other than an invalid short URL (If no value is provided, the user will see a default "404 not found" page):
 >

APPLICATION
===========

 Provide a parameter name that you will be able to use to disable tracking on specific request to short URLs (leave empty and this feature won't be enabled):
 >

 Do you want to enable a safety check which will not allow short URLs to be deleted after receiving a specific amount of visits? (yes/no) [yes]:
 >

 What is the amount of visits from which the system will not allow short URLs to be deleted? [15]:
 >

 What is the path from which shlink is going to be served? (Leave empty if you plan to serve shlink from the root of the domain) []:
 >

 Provide a comma-separated list of redis server URIs which will be used for shared caching purposes under shlink multi-instance contexts (Leave empty if you don't want to use redis cache):
 >

 Custom configuration properly generated!

Initializing database... [Running "/usr/bin/php7.4 bin/cli db:create"]   RUN  '/usr/bin/php7.4' 'bin/cli' 'db:create'
  OUT
  OUT   [OK] Database already exists. Run "db:migrate" command to make sure it is up to
  OUT        date.
  OUT
  OUT
  RES  Command ran successfully
 Success!
Updating database... [Running "/usr/bin/php7.4 bin/cli db:migrate"]   RUN  '/usr/bin/php7.4' 'bin/cli' 'db:migrate'
  OUT  Migrating database...
  OUT
  OUT   [OK] Database properly migrated!
  OUT
  OUT
  RES  Command ran successfully
 Success!
Generating proxies... [Running "/usr/bin/php7.4 vendor/doctrine/orm/bin/doctrine.php orm:generate-proxies"]   RUN  '/usr/bin/php7.4' 'vendor/doctrine/orm/bin/doctrine.php' 'orm:generate-proxies'
  OUT
  OUT   Processing entity "Shlinkio\Shlink\Core\Entity\Visit"
  OUT   Processing entity "Shlinkio\Shlink\Core\Entity\ShortUrl"
  OUT   Processing entity "Shlinkio\Shlink\Core\Entity\Domain"
  OUT   Processing entity "Shlinkio\Shlink\Core\Entity\Tag"
  OUT   Processing entity "Shlinkio\Shlink\Core\Entity\VisitLocation"
  OUT   Processing entity "Shlinkio\Shlink\Rest\Entity\ApiKey"
  OUT
  OUT   Proxy classes generated to "/var/www/vhost/shlink/web/data/proxies"
  OUT
  RES  Command ran successfully
 Success!
Clearing entities cache... [Running "/usr/bin/php7.4 vendor/doctrine/orm/bin/doctrine.php orm:clear-cache:metadata"]   RUN  '/usr/bin/php7.4' 'vendor/doctrine/orm/bin/doctrine.php' 'orm:clear-cache:metadata'
  OUT
  OUT   // Clearing all Metadata cache entries
  OUT
  OUT   [ERROR] No cache entries were deleted.
  OUT
  OUT
  RES  1 Command did not run successfully
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -36,6 +36,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com), and this 
 #### Fixed
 
 * [#729](https://github.com/shlinkio/shlink/issues/729) Fixed weird error when fetching multiple visits result sets concurrently using mariadb or mysql.
+* [#735](https://github.com/shlinkio/shlink/issues/735) Fixed error when cleaning metadata cache during installation when APCu is enabled.
 
 
 ## 2.1.3 - 2020-04-09

--- a/composer.json
+++ b/composer.json
@@ -51,7 +51,7 @@
         "shlinkio/shlink-common": "dev-master#e659cf9d9b5b3b131419e2f55f2e595f562baafc as 3.1.0",
         "shlinkio/shlink-config": "^1.0",
         "shlinkio/shlink-event-dispatcher": "^1.4",
-        "shlinkio/shlink-installer": "dev-master#508652c895fb88512b8685198be1e2cfca59f4ce as 5.0.0",
+        "shlinkio/shlink-installer": "dev-master#f51a2186cf474fb5773b0ef74b8533878de9dd1e as 5.0.0",
         "shlinkio/shlink-ip-geolocation": "^1.4",
         "symfony/console": "^5.0",
         "symfony/filesystem": "^5.0",

